### ISSUE TITLE
KHR_materials_pbrSpecularGlossiness extended model does not display properly 

### ISSUE THREAD

**Issue by cx20** [PrincessGod/CraZyPG#13]
I tried a model added to [glTF-Sample-Models](https://github.com/KhronosGroup/glTF-Sample-Models/tree/spec-gloss-vs-metal-rough/2.0/SpecGlossVsMetalRough).

However, it seems that it is not displayed properly in glTF loader of CZPG.js.

|Babylon.js|CZPG.js|
|:------:|:------:|
|[![image](https://user-images.githubusercontent.com/1833979/45266096-a321ed00-b490-11e8-8f63-20197019d4df.png)](https://cx20.github.io/gltf-test/examples/babylonjs/index.html?category=tutorialModels&model=SpecGlossVsMetalRough&scale=10&type=glTF)|[![image](https://user-images.githubusercontent.com/1833979/45266105-b92fad80-b490-11e8-9553-44dcc3edb20a.png)](https://cx20.github.io/gltf-test/examples/czpg/index.html?category=tutorialModels&model=SpecGlossVsMetalRough&scale=10&type=glTF)|

https://github.com/KhronosGroup/glTF-Sample-Models/tree/spec-gloss-vs-metal-rough/2.0/SpecGlossVsMetalRough

**Comment by PrincessGod** [comment]
You are right, but i am rebuild whole project in `rebuild` branch, when finished will merge to master !
